### PR TITLE
docs: update testing guide examples of npm scripts to support angular 7

### LINKED ISSUE #27331: Update docs for testing section to the latest Angular version (v7)
# 📚 Docs or angular.io bug report

### Description

✍️ Right now documentation for testing is outdated and npm commands are not updated to the latest angular version (v7)

## 🔬 Minimal Reproduction
Open https://angular.io/guide/testing#configure-project-for-circle-ci and you will see that npm scripts are using --single-run which is not longer supported and other outdated flags.
### What's the affected URL?**
https://angular.io/guide/testing#configure-project-for-circle-ci
https://angular.io/guide/testing#configure-project-for-travis-ci
https://angular.io/guide/testing#configure-cli-for-ci-testing-in-chrome

## 📷Screenshot

image

## 🔥 Exception or Error
<pre><code>

</code></pre>

### PR DESCRIPTION
**Pull request by mjawulski**
current documentation for setting up CI is using outdated npm scripts

PR close: #27331

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #27331

## Does this PR introduce a breaking change?

- [ ] Yes
- [ x] No

### CODE REVIEW

**Comment by trotyl** [comment]
Relates to https://github.com/angular/angular/pull/27269

**Comment by mjawulski** [comment]
> Relates to #27269

Yep I was looking for similar PR and failed to found one. I just want to mention that in my PR I have updated all occurrences of outdated npm scripts. Not only one.

**Review by Splaktar — APPROVED** [review]
LGTM

Based on https://github.com/angular/angular-cli/blob/3c875a1148f68bbd510e49451edc36061cd2baa9/packages/angular_devkit/build_angular/src/karma/schema.json#L84-L100 and https://github.com/angular/angular-cli/blob/44086c60ff1d6c26d30850bef125120f6c498ac1/packages/angular_devkit/build_angular/src/protractor/schema.json#L6-L9.

The location of the `protractor-ci-conf.js` file is also consistent with the new location of the `protractor.conf.js` file in newly generated CLI applications.

**Review by alan-agius4 — APPROVED** [review]
Thanks. LGTM.

One small nit: I’d use `no-watch` instead of `watch=false` since the other options are using this pattern.

**Comment by mjawulski** [comment]
> @mjawulski Hi. Thanks for submitting this fix. I've reached out to engineering to confirm the best technical reviewer and approver. In the meantime, could I ask you to change your commit message. "docs(docs-infra)" is now used only for change to the AIO app/infrastructure. For doc content, we now use just "docs". Thanks!
> 
> docs: update npm scripts to support angular 7

done! I have also changed --watch=false to --no-watch as suggested

**Comment by alan-agius4** [comment]
@mjawulski, can you kindly remove the merge commit, and squash all of your commits into a single one?
Thanks.

**Comment by jenniferfell** [comment]
> @mjawulski, can you kindly remove the merge commit, and squash all of your commits into a single one?
> 
> Thanks.

@alan-agius4 Good catch. 
@mjawulski After the commits are squashed, I think we can merge this. Thx.

**Review by jenniferfell — APPROVED** [review]
From a doc POV, LGTM. Trusting technical details to author and other reviewers.

File: aio/content/guide/testing.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ customLaunchers: {
 },
 ```
 
-* Create a new file, `protractor-ci.conf.js`, in the root folder of your project, which extends the original `protractor.conf.js`:
+* Create a new file, `protractor-ci.conf.js`, in the root folder of your e2e tests project, which extends the original `protractor.conf.js`:
```
Comment (jenniferfell) [inline aio/content/guide/testing.md]: The original was a bit hard to parse. I had to read this twice. Is this accurate? 

* In the root folder of your e2e tests project, create a new file named `protractor-ci.conf.js`. This file extends the original `protractor.conf.js`.

Comment (mjawulski) [reply aio/content/guide/testing.md]: I have changed paragraph, but I am afraid I don`t know how to squash the commits. Can anybody help in here?

**Comment by alan-agius4** [comment]
I'd suggest you do the following;
```
git reset --soft upstream/master
git commit aio/content/guide/testing.md -m "docs: update npm scripts to support angular 7"
git push -f
```

**Comment by mjawulski** [comment]
Done. Is everything all right now?

**Comment by mjawulski** [comment]
Done! Thanks a lot!

**Comment by alan-agius4** [comment]
All good now!

Thanks for your contribution. 

@jenniferfell this LGTM.

**Comment by alan-agius4** [comment]
Caretaker: kindly ignore buildkite failure

**Comment by mjawulski** [comment]
@jenniferfell Is anything else I can do? Or should I just wait until merged?
